app/components/provider_interface/find_candidates/safeguarding_component.rb

```diff
@@ -0,0 +1,46 @@
+class ProviderInterface::FindCandidates::SafeguardingComponent < ViewComponent::Base
+  attr_reader :application_form, :provider_user
+
+  def initialize(application_form:, provider_user:)
+    @application_form = application_form
+    @provider_user = provider_user
+  end
+
+  def rows
+    rows = [{ key: I18n.t('provider_interface.safeguarding_declaration_component.declare_safeguarding_issues'), value: declare_safeguarding_issues }]
+
+    if safeguarding_issues_declared?
+      rows << { key: I18n.t('provider_interface.safeguarding_declaration_component.safeguarding_information'), value: safeguarding_information }
+    end
+
+    rows
+  end
+
+private
+
+  def declare_safeguarding_issues
+    if safeguarding_issues_declared?
+      I18n.t('provider_interface.safeguarding_declaration_component.has_safeguarding_issues_to_declare')
+    else
+      I18n.t('provider_interface.safeguarding_declaration_component.no_safeguarding_issues_to_declare')
+    end
+  end
+
+  def safeguarding_information
+    if provider_user_can_view_safeguarding?
+      application_form.safeguarding_issues
+    else
+      I18n.t('provider_interface.safeguarding_declaration_component.cannot_see_safeguarding_information')
+    end
+  end
+
+  def provider_user_can_view_safeguarding?
+    provider_user.provider_permissions.any? do |permission|
+      permission.view_safeguarding_information == true
```

Comment:
```suggestion
      permission.view_safeguarding_information.true?
```

Just a preference, not a blocker. I think that should work.

Reply:
I don't think that works but this does

`provider_user.provider_permissions.any?(&:view_safeguarding_information)`